messaging/kafka-streams-reactive-messaging/src/test/java/io/quarkus/ts/messaging/kafka/reactive/streams/KafkaGratefulShutdownIT.java

```diff
@@ -44,36 +42,15 @@ public class KafkaGratefulShutdownIT {
     @KafkaContainer(vendor = KafkaVendor.STRIMZI)
     static KafkaService kafka = new KafkaService();
 
-    /**
-     * If topic is not created before the test, messages inside it will not be processed.
-     * for details and todo see https://github.com/quarkus-qe/quarkus-test-suite/issues/2016
-     */
-    private void createTopic(KafkaService service) throws InterruptedException {
-        Properties properties = new Properties();
-        properties.put("bootstrap.servers", service.getBootstrapUrl());
-        try (AdminClient client = AdminClient.create(properties)) {
-            client.createTopics(List.of(new NewTopic("slow", 1, (short) 1)));
-        }
-        Thread.sleep(1000);
-    }
-
     @QuarkusApplication(classes = { SlowTopicConsumer.class,
-            SlowTopicResource.class }, properties = "kafka.grateful.shutdown.application.properties")
+            SlowTopicResource.class, ReadinessObserver.class }, properties = "kafka.grateful.shutdown.application.properties")
     static RestService app = new RestService()
             .withProperty("kafka.bootstrap.servers", kafka::getBootstrapUrl)
             .withProperty("quarkus.kafka-streams.bootstrap-servers", kafka::getBootstrapUrl)
             .withProperty(KAFKA_LOG_PROPERTY, Level.DEBUG.getName());
 
     @Order(1)
     @Test
-    public void testConnection() throws InterruptedException {
```

Comment:
I am not sure we need to drop this. This smoke test can save time and effort during future investigations

Reply:
I don't mind keeping it, np.